python/lsst/ts/MTAlignment/alignmentModel.py

```diff
@@ -0,0 +1,507 @@
+
+import asyncio
+from enum import IntEnum
+import logging
+
+
+class LaserStatus(IntEnum):
+    LASERNOTCONNECTED = -1
+    LASEROFF = 0
+    LASERON = 1
+
+
+class TrackerStatus(IntEnum):
+    READY = 1
+    TWOFACE = 2
+    ADM = 3
+    DRIFT = 4
+    EMP = 5
+    ERR = 6
+
+
+class AlignmentModel():
+
+    def __init__(self, host, port):
+        self.host = host
+        self.port = port
+        self.log = logging.getLogger()
+        self.connected = False
+        self.reader = None
+        self.writer = None
+        self.first_measurement = True
+
+        self.trackerstatus = None
+        self.laserstatus = None
+        self.com_lock = asyncio.Lock()
+
+    async def connect(self):
+        """
+        Connect to the T2SA host
+        """
+        self.reader, self.writer = await asyncio.open_connection(self.host, self.port)
+        self.connected = True
+        asyncio.create_task(self.status_loop())
```

Comment:
I don't think it is a good idea to embed this kind of logic in the model. Instead of `status_loop` the model should probably have a `check_status` method instead that would return the status, instead of storing it in memory. Then, a higher level software (e.g. the CSC) can implement the status loop with proper logic in it (like going to `FAULT` if sates is `ERR`).